public/assets/boot.js

```diff
@@ -1,8 +1,8 @@
 var myCpu;
 
 term = new Terminal({
-  cols: 100,
-  rows: 20,
+  cols: 80,
```

Comment:
did you change this or did i? i was updating the cols and rows for styling

Reply:
I didnt change this, but my plan is to only merge in the line in package.json that adds the new npm dependency, and my changes to webworker.s,registerpanel.js